### DISCUSSION TITLE
Edit Tags - `rename tag, remove/add - feed from tag`

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [spacecowboy/Feeder#203] [Ideas]
Currently tag can be created only when adding a new rss feed and can only be edited when editing a rss feed setttings.

My Problem - 
- Sometime i need to rename tags because later i find a new name which suits better.
- Remove feeds from a tag
- Add feeds to a tag

Currently there is no system for dong this.

Adding this will make it earies for keeping feeds sorted.

**Comment by spacecowboy** [comment]
you can do all of it

but I agree it's clunky
